TODO:


02:32:31  lordrashmi> drafts are restricted too
02:33:14  lordrashmi> the delete thing will be a problem
02:34:25  lordrashmi> we need some way to indicate that a message has been processed


22:55:08  lordrashmi> glen_: Only thing I can think of for that mail issue is a seperate table
22:55:14  lordrashmi> and record the message ID in it
22:55:17  lordrashmi> and possible the error
23:01:52  lordrashmi> eventum_bounce
23:02:21  lordrashmi> bon_id, bon_message_id, bon_error


--- include/class.support.php	2008-06-05 23:25:40.511791440 +0300
+++ include/class.support.php	2008-06-05 23:11:57.268371086 +0300
@@ -592,9 +592,12 @@
                 if ($res != false) {
                     $return = Routing::route_notes($message);
                     if ($return === true) {
-                        Support::deleteMessage($info, $mbox, $num);
-                        return;
+                        // in case of error, create bounce, but still
+                        // delete email not to send bounce in next process :)
+                        Support::bounceMessage($email, $return);
                     }
+
+                    Support::deleteMessage($info, $mbox, $num);
                     return;
                 }
             }
